src/hypofuzz/hypofuzz.py

```diff
@@ -100,6 +104,14 @@ def _should_trace(self) -> bool:
         return False
 
 
+# like choices_size, but handles ChoiceTemplate
+def _choices_size(choices: tuple[Union[ChoiceT, ChoiceTemplate], ...]) -> int:
+    return sum(
+        1 if isinstance(choice, ChoiceTemplate) else choices_size([choice])
+        for choice in choices
+    )
```

Comment:
Maybe we should have something like that upstream in Hypothesis?  Certainly doesn't block this PR though.

Reply:
possibly, and similarly for `choices_key` etc. I'm wary of locking in these semantics in hypothesis, since nothing there needs this right now. (But maybe in the future)